readthedocs/rtd_tests/tests/test_api.py

```diff
@@ -1700,16 +1703,18 @@ def setUp(self):
             verbose_name="master",
             active=True,
             project=self.project,
+            type=BRANCH,
         )
         self.version_tag = get(
             Version,
             slug="v1.0",
             verbose_name="v1.0",
             active=True,
             project=self.project,
+            type=TAG,
         )
         self.github_payload = {
-            "ref": "master",
+            "ref": "refs/heads/master",
```

Comment:
`refs/heads/master` is the real value that GitHub will send us?

Reply:
yep